data/schema/units/modifications.cfg

```diff
@@ -217,6 +217,8 @@
 	{SIMPLE_KEY male_name t_string}
 	{SIMPLE_KEY female_name t_string}
 	{SIMPLE_KEY description t_string}
+	{SIMPLE_KEY exclude_traits string_list}
+	{SIMPLE_KEY require_traits string_list}
```

Comment:
My bad, the units should not have that keyword, it was wrongfully still in the unit file, I'll revert the change. The feature is strictly trait-macro bound, the units should not notice anything.